lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -1,4 +1,4 @@
-﻿//-------------------------------------------------------------------------------------------------------
+//-------------------------------------------------------------------------------------------------------
```

Comment:
What happened here? Was there a byte order marker?

Reply:
Yes - oddly this was the only file in the source tree with one.